akka-javasdk/src/main/scala/akka/javasdk/impl/SdkRunner.scala

```diff
@@ -546,6 +558,7 @@ private final class Sdk(
   private val viewDescriptors: Seq[SpiViewDescriptor] =
     componentClasses
       .filter(hasComponentId)
+      .filter(isNotExcluded)
       .collect {
         case clz if classOf[View].isAssignableFrom(clz) => ViewDescriptorFactory(clz, serializer, sdkExecutionContext)
       }
```

Comment:
what about HttpEndpoints? shouldn't those also be possible to exclude?

Reply:
good point, added filtering there